core/dbt/events/types.py

```diff
@@ -266,7 +266,7 @@ def message(self) -> str:
 
 @dataclass
 class SystemReportReturnCode(DebugLevel, Cli, File):
-    code: int
+    returncode: int
 
     def message(self) -> str:
         return f"command return code={self.code}"
```

Comment:
Should use the renamed version - `self.returncode`

Reply:
yes! thank you.